### PR TITLE
Run vagrant-spec tests

### PR DESCRIPTION
**Pull request by soapy1**

### CODE REVIEW

File: .ci/spec/clean-packet.sh [hunk 1]
```diff
@@ -4,18 +4,28 @@ csource="${BASH_SOURCE[0]}"
...
+. "${root}/.ci/spec/env.sh"
 
 pushd "${root}" > "${output}"
 
 echo "Cleaning up packet device..."
 
+unset PACKET_EXEC_PERSIST
+unset PACKET_EXEC_PRE_BUILTINS
+
+export PACKET_EXEC_REMOTE_DIRECTORY="run-${GITHUB_RUN_ID}"
```
Comment (soapy1) [inline .ci/spec/clean-packet.sh:17]: For some reason, by the time that the ci gets to this part the `job_id` changes from `run-${GITHUB_RUN_ID}` to `run1-${GITHUB_RUN_ID}`. This forces it back.

**Review by briancain — DISMISSED** [review]
Approving just cuz I am excited to see this happen! :tada: Thanks for finishing it out. :pray: :chart_with_upwards_trend:

**Review by chrisroberts — CHANGES_REQUESTED** [review]

File: .ci/spec/run-test.sh [hunk 2]
```diff
@@ -4,31 +4,27 @@ csource="${BASH_SOURCE[0]}"
...
 . "${root}/.ci/common.sh"
+. "${root}/.ci/spec/env.sh"
 
 pushd "${root}" > "${output}"
 
 # Assumes packet is already set up
-
-# job_id is provided by common.sh
-export PACKET_EXEC_REMOTE_DIRECTORY="${job_id}"
+unset PACKET_EXEC_PRE_BUILTINS
```
Comment (soapy1) [inline .ci/spec/run-test.sh:13]: When this is set, it re-installs all the vmware tools and causes `vagrant --provision` to fail

Comment (chrisroberts) [reply .ci/spec/run-test.sh:13]: :+1: though we should have a look at that helper and see why it's not a no-op at this point.

**Review by chrisroberts — APPROVED** [review]
:tada:

**Pull request merged**